metricbeat/module/system/uptime/_meta/data.json

```diff
@@ -0,0 +1,18 @@
+{
+    "@timestamp": "2016-06-29T19:25:08.155Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "metricset": {
+        "module": "system",
+        "name": "uptime",
+        "rtt": 33
+    },
+    "system": {
+        "uptime": {
+            "uptime_ms": 789306155
```

Comment:
We generally do `uptime.ms` in this case (not with actual dot, but as a sub-doc).